cmd/state-installer/cmd.go

```diff
@@ -300,23 +305,23 @@ func execute(out output.Outputer, cfg *config.Instance, an analytics.Dispatcher,
 		return postInstallEvents(out, cfg, an, params, isUpdate)
 	}
 
-	// Check if state tool already installed
-	if !params.force && stateToolInstalled {
-		logging.Debug("Cancelling out because State Tool is already installed")
-		out.Print(fmt.Sprintf("State Tool Package Manager is already installed at [NOTICE]%s[/RESET]. To reinstall use the [ACTIONABLE]--force[/RESET] flag.", installPath))
-		an.Event(AnalyticsFunnelCat, "already-installed")
-		return postInstallEvents(out, cfg, an, params, true)
-	}
+	storeInstallSource(params.sourceInstaller)
```

Comment:
This doesn't seem to get called under successful conditions.

Reply:
You're correct. I misunderstood the meaning/purpose of this function.